code/dynamic_programming/src/fibonacci/fibonacci.cpp

```diff
@@ -1,28 +1,42 @@
-from sys import argv
+/* Part of Cosmos by OpenGenus Foundation */
 
+#include <string>
+#include <sstream>
+#include <iostream>
 
-def fibonacci(n):
-    """
-    N'th fibonacci number using Dynamic Programming
-    """
-    arr = [0, 1]  # 1st two numbers as 0 and 1
+using namespace std;
 
-    while len(arr) < n + 1:
-        arr.append(0)
+int dynamic_programming(int num);
 
-    if n <= 1:
-        return n
-    else:
-        if arr[n - 1] == 0:
-            arr[n - 1] = fibonacci(n - 1)
-        if arr[n - 2] == 0:
-            arr[n - 2] = fibonacci(n - 2)
+int main(){
 
-    arr[n] = arr[n - 2] + arr[n - 1]
-    return arr[n]
+  int number;
 
+  cout << "Please enter an integer: ";
```

Comment:
Make this input prompt string more specific so that the user could understand in the first look itself what this input is for. Like say:-
```
"Enter the value of n: "   
or   
 "Enter the limit for the fibonacci series: "
```